### PR TITLE
Use remote registry API [prax]

### PR DESCRIPTION
**Pull request by grod220**
Follow up after https://github.com/penumbra-zone/web/pull/1472

Updates packages so that Prax can use the new registry API that makes frontend + rpc list dynamic (versus bundled). See explanation in previous PR.

### CODE REVIEW

File: packages/ui/components/ui/tx/index.tsx
Comment (grod220) [inline packages/ui/components/ui/tx/index.tsx:1]: Lifted 1:1 from: https://github.com/penumbra-zone/web/pull/1472/files#diff-69fa906c1be5254320e41e6115e025eb53b41cccfdb26baff5fd367736317c2d. So much redundancy...

File: apps/extension/src/shared/components/loading-list.tsx
Comment (grod220) [inline apps/extension/src/shared/components/loading-list.tsx:1]: Screenshot 2024-07-15 at 10 11 17 AM

File: apps/extension/src/hooks/numeraires-query.ts
Comment (grod220) [inline apps/extension/src/hooks/numeraires-query.ts:1]: Hook version of https://github.com/prax-wallet/web/pull/100/files#diff-b189ef054359c4f7abf8e7a52d29aa1f006438ced433de671cf8742a6234a45e

**Review by Valentine1898 — APPROVED** [review]
LGTM!

**Pull request merged**